.circleci/config.yml

```diff
@@ -54,7 +54,7 @@ commands:
       - run:
           name: "Install Grafana build pipeline tool"
           command: |
-            VERSION=0.2.12
+            VERSION=0.4.0
```

Comment:
Is this change intentional?

Reply:
@dprokop Yes, I uploaded a preliminary version of the build pipeline tool that reads these URLs from Grafana's package.json.